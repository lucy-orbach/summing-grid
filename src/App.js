import React, { Component } from 'react';
import 'App.modules.css';
import Header from 'components/header/Header';
import SumWidgetContainer from 'components/sumWidget/SumWidgetContainer'

const App = () => (
  <div className="App" data-test="app" >
    <Header />
    <main>
      <section>
        <SumWidgetContainer />
      </section>
    </main>
  </div>
);

export default App;
