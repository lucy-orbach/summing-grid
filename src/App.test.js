import React from 'react';
import 'setupTests';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from 'App';

describe('<App />', () => {

	it('should render App component ', () => {
		const wrapper = shallow(<App />);
		expect( wrapper.find('[data-test="app"]').exists()).toBe(true);
	});

	it('matches the snapshot', () => {
		const tree = shallow(<App />);
		expect(toJson(tree)).toMatchSnapshot()
	});

});

