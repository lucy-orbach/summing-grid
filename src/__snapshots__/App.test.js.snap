// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches the snapshot 1`] = `
<div
  className="app"
  data-test="app"
>
  <Header />
  <main
    className="main_container"
  >
    <ErrorBoundary>
      <SumWidgetContainer />
    </ErrorBoundary>
  </main>
</div>
`;
